tests/integration/api/queries/accountCartByAccountId/accountCartByAccountId.test.js

```diff
@@ -0,0 +1,72 @@
+import encodeOpaqueId from "@reactioncommerce/api-utils/encodeOpaqueId.js";
+import importAsString from "@reactioncommerce/api-utils/importAsString.js";
+import Factory from "/tests/util/factory.js";
+import TestApp from "/tests/util/TestApp.js";
+
+const AccountCartByAccountIdQuery = importAsString("./AccountCartByAccountIdQuery.graphql");
+
+jest.setTimeout(300000);
+
+let accountCartByAccountIdQuery;
+let mockCart;
+let mockCustomerAccount;
+let opaqueAccountId;
+let shopId;
+let testApp;
+
+beforeAll(async () => {
+  testApp = new TestApp();
+  await testApp.start();
+  shopId = await testApp.insertPrimaryShop();
+  accountCartByAccountIdQuery = testApp.query(AccountCartByAccountIdQuery);
+
+  // create mock customer account
+  mockCustomerAccount = Factory.Account.makeOne({
+    _id: "mockCustomerAccountId",
+    roles: {
+      [shopId]: ["owner"]
```

Comment:
Is this a mock customer account or admin because you added the `owner` permission? Leaving it as a blank array should be fine.

Reply:
@mikemurray I have removed the unnecessary role and ready for another review.